src/main/java/com/google/sps/servlets/TasksServlet.java

```diff
@@ -34,8 +32,7 @@
  * common functions (Issue #26)
  */
 @WebServlet("/tasks")
-public class TasksServlet extends HttpServlet {
-
+public class TasksServlet extends AuthenticatedHttpServlet {
   /**
    * Returns taskNames from the user's Tasks account
```

Comment:
I know this change was not made in this PR, but is this description exact? The json string in the response contains more than task titles

Reply:
As of now (in the prototype version) - that's all it did. During the MVP this will be changed to be a bit more feature rich